ipaserver/install/adtrustinstance.py

```diff
@@ -345,9 +345,14 @@ def __add_rid_bases(self):
 
             # Abort if RID bases are too close
             local_range = ranges_with_no_rid_base[0]
-            size = local_range.single_value.get('ipaIDRangeSize')
+            try:
+                size = int(local_range.single_value.get('ipaIDRangeSize'))
```

Comment:
I don't know what are the implications of `ipaIDRangeSize` missing or whether we'd even get here if it was not here.
I am judging that it's a hard requirement, `int(None)` will lead to the failure nonetheless, I suppose it's OK to hard fail here. @abbra might have a different opinion.